Classes/Tree/NodePathBuilder.php

```diff
@@ -109,10 +115,15 @@ public function getPathFromRootToNode($nodeUid, $startIndex = 0, $length = 1000)
      * @param integer $length
      * @return array nodes as plain array
      */
-    public function getPathFromNodeToRoot($nodeUid, $startIndex = 0, $length = 1000)
+    public function getPathFromNodeToRoot(int $nodeUid, int $startIndex = 0, int $length = 1000):array
     {
+        if($nodeUid === 0) {
```

Comment:
Das löst das Problem IMHO nicht. 
Es ist doch sicher zu stellen, dass das Ergebnis von `$this->buildPathFromNodeToRoot($nodeUid)` immer ein Array ist und wenn das nicht möglich ist, das Ergebnis zu prüfen und dann nich in array_slice auszuführen.